libs/tkwsm/test/SolvingProblems/test_RandomGraphs.cpp

```diff
@@ -376,11 +376,11 @@ SCENARIO(
     entry = -1;
   }
   TestParameters params;
-  params.timeout = 30000;
+  params.timeout = 60000;
```

Comment:
Why is this changed? Is the interaction longer?

Reply:
I found I needed to increase it because the coverage test was failing with a timeout here. (It was only slightly over 30000 but I changed it to 60000 to be on the safe side.)

Reply:
https://github.com/CQCL/tket/runs/7524375317

Reply:
Perfect, thank you!